apps/web/src/components/SortableField.tsx

```diff
@@ -0,0 +1,22 @@
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+import { Field, mapFieldToCroatian } from '@internship-app/types';
+
+type Props = {
+  field: Field;
+};
+
+export const SortableField: React.FC<Props> = ({ field }) => {
```

Comment:
Šteta što si samo ograničio da je sortable za discipline internshipa

Reply:
to je jedino misto di se koristi u appu pa to (bar za sad) ne bi minja